pyproject.toml

```diff
@@ -0,0 +1,8 @@
+[tool.mypy]
```

Comment:
Somewhere here, show_error_codes is being caught as deprecated. However, since it's not being used, and it's a minor issues, I think we let this go.